include/ceed/ceed.h

```diff
@@ -596,6 +606,8 @@ CEED_EXTERN int CeedQFunctionContextGetData(CeedQFunctionContext ctx,
     CeedMemType mem_type, void *data);
 CEED_EXTERN int CeedQFunctionContextRestoreData(CeedQFunctionContext ctx,
     void *data);
+CEED_EXTERN int CeedQFunctionContextGetContextSize(CeedQFunctionContext ctx,
+    size_t *ctx_size);
```

Comment:
This declaration is also found in `backend.h`, which confuses the Julia bindings

Reply:
Whoops, sorry about that -- I definitely forgot to delete from `backend.h` after adding to `ceed.h`.  
Are you planning to eventually bring over the commits from your branch to this one (meaning I don't need to go ahead and make this change here as well)?

Reply:
I would suggest making the change here. Eventually I'll merge the changes from the Julia branch, but I am experimenting with some different options, so it might not happen immediately.

Reply:
Sounds good, I fixed it in 38c8540f60fa93923871da0dbdba0fe187fb12bf